pkg/api/kyverno/v1/types.go

```diff
@@ -224,8 +224,8 @@ type Deny struct {
 type Generation struct {
 	ResourceSpec
 	Synchronize bool        `json:"synchronize,omitempty"`
-	Data        interface{} `json:"data,omitempty"`
-	Clone       CloneFrom   `json:"clone,omitempty"`
+	Data        []interface{} `json:"data,omitempty"`
+	Clone       []CloneFrom   `json:"clone,omitempty"`
```

Comment:
@evalsocket this will break compatibility. To introduce such a change, we need to use a new API version and migrate existing resources.